lib/plugins/aws/package/compile/events/apiGateway/lib/requestValidator.test.js

```diff
@@ -0,0 +1,449 @@
+'use strict';
+
+const chai = require('chai');
+const runServerless = require('../../../../../../../../tests/utils/run-serverless');
+const fixtures = require('../../../../../../../../tests/fixtures');
+
+const expect = chai.expect;
+chai.use(require('chai-as-promised'));
+
+describe('#compileRequestValidators()', () => {
+  after(fixtures.cleanup);
+
+  describe(' reusable schemas ', () => {
+    it('Should process schema from apiGateway provider, full config', () =>
+      fixtures.extend('requestSchema', {}).then(fixturePath =>
```

Comment:
We do not extend fixture with anything, so let's not use `fixture.extend`

Reply:
Im really running that to help populate the serverless block using the fixturePath. I didn't see another example just using `runServerless`, but I will work through it.

Reply:
Here's an example of using fixture directly: https://github.com/serverless/serverless/blob/73107822945a878abbdebe2309e8e9d87cc2858a/lib/plugins/aws/package/compile/functions/index.test.js#L2629-L2632